### PR TITLE
CT-1642 Fix support for GEOMETRY and GEOGRAPHY for incremental load

### PR DESCRIPTION
**Pull request by zajca**
Jira: CT-1642
Connection PR: XXX
SAPI PR: XXX

---

`getUpdateWithPkCommand` now can compare GEOMETRY and GEOGRAPHY datatypes, previously it was wrong and query would fail as DISTINCT FROM cannot compare two same types.
Not it will cast them to EWKT and compare with DISTINCT FROM

### CODE REVIEW

**Comment by romanbracinik** [comment]
👁️

File: packages/php-db-import-export/tests/functional/Snowflake/ToFinal/SqlBuilderTest.php [hunk 1]
```diff
@@ -1621,6 +1622,358 @@ public function testGetUpdateWithPkCommandNullManipulationWithTimestampFeatures(
...
+                    ),
+                ),
+                new SnowflakeColumn(
+                    'geography',
+                    new Snowflake(
+                        Snowflake::TYPE_GEOGRAPHY,
+                        ['nullable' => true],
+                    ),
+                ),
+            ]),
```
Comment (romanbracinik) [inline packages/php-db-import-export/tests/functional/Snowflake/ToFinal/SqlBuilderTest.php:1701]: Pozeral som a vsade pouzivame nejake genericke stlpce pre stagin tabulku napr pouzivame na to `createNullableGenericColumn`, takze staging tabulka je stale varchar nie? Potom tento test je trochu zavadzajuci ze ma typy.

**Review by romanbracinik — COMMENTED** [review]
niesom si isty tou stage tabulkou v testoch

File: packages/php-db-import-export/tests/functional/Snowflake/ToFinal/SqlBuilderTest.php (hunk 1 above)
Comment (zajca) [reply packages/php-db-import-export/tests/functional/Snowflake/ToFinal/SqlBuilderTest.php:1701]: To sa právě změní https://github.com/keboola/connection/pull/5181 popisuju to v tom jira ticketu

Comment (romanbracinik) [reply packages/php-db-import-export/tests/functional/Snowflake/ToFinal/SqlBuilderTest.php:1701]: Jo to som zaznamenal, zesa to ma menit akurat som nevedel ze uz tu, tak asi ok ;)

**Review by romanbracinik — APPROVED** [review]
🙆‍♂️

**Pull request merged**